src/Uno.Extensions.Core.Generators/PropertySelector/PropertySelectorsGenerationTool.cs

```diff
@@ -14,49 +13,35 @@ internal class PropertySelectorsGenerationTool : ICodeGenTool
 	/// <inheritdoc />
 	public string Version => "1";
```

Comment:
Not sure if I should increment this?

Reply:
We could, but it's only for the `[GeneratedCode]` and other generated headers ... I don't think it has a significant impact